lib/registries/addon/components/responsive-dropdown/template.hbs

```diff
@@ -0,0 +1,15 @@
+{{#basic-dropdown
+    verticalPosition='below'
+    horizontalPosition='right'
+    calculatePosition=this.calculatePosition
+    disabled=@disabled
+    onClose=(action 'enableScroll')
+    as |dd|}}
+
+    {{yield (hash trigger=dd.trigger)}}
```

Comment:
It looks like you might be able to use one `{{yield}}` here?

disclaimer: this is a guess, I have not tried this.

```hbs
{{yield (hash
    trigger=dd.trigger
    content=(component dd.content overlay=this.useOverlay local-class=(if this.useOverlay 'reg-modal-dialog'))
))}}
```